### PR TITLE
Added location CRUD endpoints

### PR DESCRIPTION
**Pull request by TaylorBelk1**

### CODE REVIEW

File: api/locations/locationsRoutes.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        })
+});
+
+router.get('/:id', (req, res) => {
+    Place.getPlaceById(req.params.id)
+        .then(place => {
+            if(place) {
+                res.status(200).json(place)
+            } else {
+                res.status(404).json('Error: Could not find that location')
```
Comment (BoxOfCereal) [inline api/locations/locationsRoutes.js:26]: Looks like a small typo? Is this the structure? `{e: 'e message'}`

Comment (TaylorBelk1) [reply api/locations/locationsRoutes.js:26]: Wait where? Lol which line

Comment (BoxOfCereal) [reply api/locations/locationsRoutes.js:26]: Line 26. Sorry that's probably nit picking.

**Review by BoxOfCereal — APPROVED** [review]

File: api/locations/locationsRoutes.js (hunk 1 above)
Comment (TaylorBelk1) [reply api/locations/locationsRoutes.js:26]: oh no, I did that before the catch. To test if there was a "place" if not then I spit out you didn't put in a place, any status 500 errors will show in the catch on line 28

**Pull request merged**